test/integration/ctl/ctl_create_cr_test.go

```diff
@@ -147,6 +141,9 @@ func TestCtlCreateCR(t *testing.T) {
 		t.Run(name, func(t *testing.T) {
 			streams, _, _, _ := genericclioptions.NewTestIOStreams()
 
+			cleanUpFunc := setupPathForTest(t)
+			defer cleanUpFunc()
```

Comment:
Nice 😄  this is way cleaner! Removes all traces/mention of temporary directories in here too 😄